static/panes/cfg-view.ts

```diff
@@ -77,6 +79,33 @@ function attrs(attributes: Record<string, string | number | null>) {
         .join(' ');
 }
 
+function special_round(x: number) {
```

Comment:
I would not be surprised if there is a better way to do this and I am greatly overcomplicating things 😅